taxi/templates/taxi/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html"%}

```

Comment:
Consider adding a space after `{% extends "base.html" %}` to maintain consistency with other block tags like `{% block content %}`.